.travis.yml

```diff
@@ -112,6 +112,7 @@ matrix:
       os: osx
       osx_image: xcode9.2
     - name: "Windows Server, 1809"
+      # runs on all branches
```

Comment:
Is this relevant to this PR? Or is this one of those comments that's too small to create another PR for, so it's best to just bundle it into this one?

Reply:
Yeah I'm slipping it in. Look it's even got its own commit message, but it didn't seem worth another PR.